apis/falcon/v1alpha1/falconcontainer_types.go

```diff
@@ -52,6 +52,8 @@ type RegistrySpec struct {
 	TLS RegistryTLSSpec `json:"tls,omitempty"`
 	// Azure Container Registry Name represents the name of the ACR for the Falcon Container push. Only applicable to Azure cloud.
 	AcrName *string `json:"acr_name,omitempty"`
+	// EcrIamRoleArnForInjector allows injector to assume AIM role to access registry
+	EcrIamRoleArnForInjector *string `json:"ecr_iam_role_arn_for_injector,omitempty"`
```

Comment:
Can we make the json name easier e.g. `ecr_iam_arn`

Reply:
what about `ecr_iam_role_arn`

There is a difference between role arn and policy arn and users will definitely stumble upon this.

Reply:
Sounds good to me.

Reply:
Thank You from me.

Reply:
hahaha